src/components/SubmitTokenInfo/ImgUpload.tsx

```diff
@@ -0,0 +1,61 @@
+import classNames from 'classnames'
+import { useRef } from 'react'
+import UploadIcon from '../../assets/arrow_up_circle.png'
+import DeleteIcon from '../../assets/delete.png'
+import styles from './ImgUpload.module.scss'
+
+type Props = {
+  value: string | null
+  label: string
+  onChange: React.ChangeEventHandler<HTMLInputElement>
+  onClear: () => void
+  labelRightAddon?: React.ReactNode
+  placeholder?: string
+  isRequired?: boolean
+  className?: string
+}
+
+export const ImgUpload = (prop: Props) => {
+  const { value, label, onChange, labelRightAddon, placeholder, isRequired, className, onClear } = prop
+  const inputRef = useRef<HTMLInputElement>(null)
+
+  const handleClick = () => {
+    inputRef.current?.click()
+  }
+  // clear ref value on click to trigger onChange event event if choose the same file of last choice
+  const clearValue = () => {
+    inputRef.current!.value = ''
+  }
+  return (
+    <div className={classNames(styles.container, className)}>
+      <div className={styles.label}>
+        {label} {isRequired && <span className={styles.requiredIcon}>*</span>}
+        {labelRightAddon && <span className={styles.labelRightAddon}>{labelRightAddon}</span>}
+      </div>
+
+      <input
+        type="file"
+        onChange={onChange}
+        className={styles.input}
+        ref={inputRef}
+        accept="image/*"
+        onClick={clearValue}
+      />
+      <button type="button" className={styles.inputWrapper} onClick={value ? undefined : handleClick}>
```

Comment:
use `label` element with `for` property to match the `input`

Ref:
- label: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label